create-keystone-next-app/src/index.ts

```diff
@@ -106,14 +102,14 @@ const installDeps = async (cwd: string): Promise<'yarn' | 'npm'> => {
   await fs.mkdir(normalizedArgs.directory);
   await Promise.all([
     ...[
-      '.gitignore',
+      '_gitignore',
       'schema.ts',
       'package.json',
       'tsconfig.json',
     ].map((filename) =>
       fs.copyFile(
         path.join(starterDir, filename),
-        path.join(normalizedArgs.directory, filename)
+        path.join(normalizedArgs.directory, filename.replace('_', '.'))
```

Comment:
Wouldn't this also rename files like `foo_bar` -> `foo.bar`?

Perhaps only target replacements at the start of the filename:

```suggestion
        path.join(normalizedArgs.directory, filename.replace(/^_/, '.'))
```

Reply:
I was keenly aware of this but since the names are literally hard-coded like two lines above I figured it would be fine?

Reply:
Yeah, probably would be. I'm just conscious of the potential foot-gun in the future if we ever refactored this code, etc.